### PR TITLE
[hpal] add judgment to iol usage graph + mana gain from divine revelations

### PR DESCRIPTION
**Pull request by mfield**
test logs:
- https://www.warcraftlogs.com/reports/qZrTGNd8C4DPbzKm#fight=last&type=healing&source=19
image
image
- https://www.warcraftlogs.com/reports/m3v6Z8DrdK7MtVzb#fight=23&type=healing&source=15
image
image

### CODE REVIEW

File: src/analysis/retail/paladin/holy/modules/apl.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import TALENTS from 'common/TALENTS/paladin';
+import * as cnd from 'parser/shared/metrics/apl/conditions';
+
+import { AnyEvent } from 'parser/core/Events';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+
+const inWings = cnd.buffPresent(TALENTS.AVENGING_CRUSADER_TALENT);
+
+const APL: Rule[] = [
```
Comment (abelito75) [inline src/analysis/retail/paladin/holy/modules/apl.tsx]: HoW doesn't want an APL for holy paladin in wowA

**Review by abelito75 — CHANGES_REQUESTED** [review]
HoW doesn't want an APL for healing specs

File: src/analysis/retail/paladin/holy/modules/apl.tsx (hunk 1 above)
Comment (mfield) [reply src/analysis/retail/paladin/holy/modules/apl.tsx]: oh accidental inclusion, will delete. was playing with this when AC was the build, not even relevant now.

Comment (mfield) [reply src/analysis/retail/paladin/holy/modules/apl.tsx]: but FWIW i think the MW one adds value

**Review by abelito75 — APPROVED** [review]

**Pull request merged**